SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetConnection.h

```diff
@@ -62,4 +64,9 @@ class SPATIALGDK_API USpatialNetConnection : public UIpConnection
 	// Player lifecycle
 	Worker_EntityId PlayerControllerEntity;
 	FTimerHandle HeartbeatTimer;
+
+	ULatencyManager* LatencyManager;
```

Comment:
Why is this a pointer instead of an instance?

Reply:
see below comment